### ISSUE TITLE
QBtn: Missing aria-label on rendered button

### ISSUE THREAD

**Issue by petronellius** [quasarframework/quasar#5549]
**Describe the bug**
QBtn with icon and without label used together with QTooltip does not generate aria-label.
The bellow code
```html
<q-btn
	@click="close"
	flat
	round
	dense
	icon="mdi-close">
	<q-tooltip>{{ $t(translationKeys.controls.buttons.close) }} </q-tooltip>
</q-btn>
```
is rendered as
```html
<button data-v-68b306d8="" tabindex="0" type="button" class="q-btn inline q-btn-item non-selectable no-outline q-btn--flat q-btn--round q-focusable q-hoverable q-btn--dense" style="font-size: 10px;">
```
And Chrome dev tools audit shows accessibility error "Buttons do not have an accessible name"

**Expected behavior**
Rendered button will have aria-label="Close".

**Comment by pdanpdan** [comment]
I agree that aria is not yet fully done. But how should anybody guess the aria-label if you don't specify it?

**Comment by rstoenescu** [comment]
It's the developer's job to add the aria-label attribute. Quasar cannot guess what to fill aria-label with...

**Comment by petronellius** [comment]
I would expect that button aria-label will have same value as q-tooltip has. 
According [the wai-aria specification](https://www.w3.org/TR/wai-aria-1.1/#aria-label) aria-label should be used when a visible label or visible tooltip (title attribute) is undesirable. So aria-label is intended to be used as alternative to tooltip (title attribute).

**Comment by rstoenescu** [comment]
QBtn and QTooltip are two separate components. Checking if a QBtn has a QTooltip introduces serious overhead, which we do not want. Would also make it impossible for handling SSR without hydration errors.
